test/integration/scenarios/03.basicWithNsDelete/verify/manifest_test.go

```diff
@@ -304,6 +304,51 @@ func TestBasic_InstanceDeletion(t *testing.T) {
 	require.Equal(t, diff, 1, "there should be one namespace difference")
 }
 
+func TestBasic_VerifySameInstanceRecreationInNamespace(t *testing.T) {
+	// Manifests to deploy
+	var testManifests = []string{
+		"../manifest/oss/solution2.yaml",
+		"../manifest/oss/target2.yaml",
+		"../manifest/oss/instance-recreate.yaml",
+	}
+
+	// Deploy the manifests
+	for _, manifest := range testManifests {
+		fullPath, err := filepath.Abs(manifest)
+		require.NoError(t, err)
+
+		err = shellcmd.Command(fmt.Sprintf("kubectl apply -f %s -n default", fullPath)).Run()
+		require.NoError(t, err)
+	}
+
+	cfg, err := testhelpers.RestConfig()
+	require.NoError(t, err)
+	dyn, err := dynamic.NewForConfig(cfg)
+	require.NoError(t, err)
+
+	// Verify new instance status
+	for {
+		resources, err := dyn.Resource(schema.GroupVersionResource{
+			Group:    "solution.symphony",
+			Version:  "v1",
+			Resource: "instances",
+		}).Namespace("default").List(context.Background(), metav1.ListOptions{})
+		require.NoError(t, err)
+
+		require.Len(t, resources.Items, 1, "there should be only one instance")
+
+		status := getStatus(resources.Items[0])
+		fmt.Printf("Current instance status: %s\n", status)
```

Comment:
We need to verify the status details. Cleaning up summary is to make sure the previous status won't be leaked after recreation.